framework/requirements.txt

```diff
@@ -1,61 +1,62 @@
-aiohttp==3.6.2
 aiohttp-cache==2.0.0
-aiohttp-jinja2==1.2.0
 aiohttp-cors==0.7.0
+aiohttp-jinja2==1.2.0
 aiohttp-swagger==1.0.14
-asn1crypto==0.24.0
-azure-common==1.1.23
-azure-storage-blob==1.4.0
-azure-storage-common==1.4.2
-boto3==1.9.82
-botocore==1.12.240
-cachetools==3.1.1
-certifi==2019.9.11
-cffi==1.12.3
+aiohttp==3.6.2
+asn1crypto==1.3.0
+azure-common==1.1.25
+azure_core==1.5.0
+azure-storage-blob==12.3.1
+azure-storage-common==2.1.0
+boto3==1.13.1
+botocore==1.16.1
+cachetools==4.1.0
+certifi==2020.4.5.1
+cffi==1.14.0
 chardet==3.0.4
 Click==7.0
 clickclick==1.2.2
 connexion==2.6.0
-cryptography==2.6.1
+cryptography==2.9.2
 defusedxml==0.6.0
-docker==3.7.0
 docker-pycreds==0.4.0
+docker==4.2.0
 docutils==0.15.2
-google-api-core==1.14.2
-google-auth==1.6.3
-google-cloud-pubsub==1.0.2
-googleapis-common-protos==1.6.0
-grpc-google-iam-v1==0.12.3
-grpcio==1.24.0
 ecdsa==0.13
 future==0.17.1
-idna==2.8
+google-api-core==1.16.0
+google-auth==1.14.1
+google-cloud-pubsub==1.4.3
+googleapis-common-protos==1.51.0
+grpc-google-iam-v1==0.12.3
+grpcio==1.28.1
+idna==2.9
 inflection==0.3.1
 itsdangerous==1.1.0
 Jinja2==2.10.1
-jmespath==0.9.4
+jmespath==0.9.5
 jsonschema==2.6.0
 MarkupSafe==1.1.1
-openapi-spec-validator==0.2.6
 more-itertools==8.2.0
-protobuf==3.9.2
-pyasn1==0.4.7
-pyasn1-modules==0.2.6
+openapi-spec-validator==0.2.6
 pathlib==1.0.1
+protobuf==3.11.3
 psutil==5.7.0
-pycparser==2.19
-python-dateutil==2.8.0
+pyasn1-modules==0.2.8
+pyasn1==0.4.8
+pycparser==2.20
+python-dateutil==2.8.1
 python-jose==3.0.1
-pytz==2018.9
-requests==2.21.0
-setuptools==40.6.2
+pytz==2020.1
+requests==2.23.0
 rsa==4.0
-s3transfer==0.1.13
-six==1.12.0
+s3transfer==0.3.3
+setuptools==46.1.3
+six==1.14.0
 SQLAlchemy==1.3.11
 swagger-ui-bundle==0.0.3
-urllib3==1.24.3
-uvloop==0.11.3
-websocket-client==0.56.0
+urllib3==1.25.9
+uvloop==0.14.0
+websocket-client==0.57.0
 Werkzeug==0.15.2
```

Comment:
```suggestion
Werkzeug==1.0.1
```

Reply:
Done in https://github.com/wazuh/wazuh/pull/5150/commits/fb4d4a4025418844084983122ee7f3a7d5a7f6ff and https://github.com/wazuh/wazuh/pull/5150/commits/3fd9f04ba5d2ab212f4e4a15907b9da45b9adf3d